anomalib/models/cflow/config.yaml

```diff
@@ -62,6 +62,9 @@ logging:
   logger: [] # options: [tensorboard, wandb, csv] or combinations.
   log_graph: false # Logs the model graph to respective logger.
 
+optimization:
+  export_mode: ""
```

Comment:
can we have `null` value here? and a comment showing the options? Something like:

```yaml
optimization:
   export_mode: null    # options: onnx, openvino
```